nominatim/clicmd/admin.py

```diff
@@ -41,6 +41,8 @@ def add_args(self, parser: argparse.ArgumentParser) -> None:
                           help='Print performance analysis of the indexing process')
         objs.add_argument('--collect-os-info', action="store_true",
                           help="Generate a report about the host system information")
+        objs.add_argument('--clean-deleted', action='store_true',
+                          help='Clean up deleted relations')
```

Comment:
If you want you can simply make the age a required argument to `--clean-deleted`. Then argparse ensures that the the argument is present and the code can be a lot simpler.

Reply:
Thank you for your feedback @lonvia! I wasn't able to make the age required directly because it was causing errors with the other commands that didn't include age. Instead I decided to raise a parser error right in the `clicmd/admin.py` file:
`if not args.age:
                self.parser.error('Age is required for --clean-deleted command')`
I moved the test for that into the `cli/test_cmd_admin.py` file. I think the solution is a bit cleaner, but if you know a way to make the argument required directly I'd be happy to switch to that instead.

Reply:
I meant to give clean-deleted an argument:

```
objs.add_argument('--clean-deleted', action='store', metavar='AGE', help='...')

...

age = args.clean_deleted

...
```

But the current solution works as well. Just one tiny nitpick: simply do `raise UsageError()` instead of using `self.parser.error()`. Just for consistency with the rest of the code.

Reply:
Thank you for clarifying lonvia. I misunderstood where you were going with that. I like your solution better. I've made the required changes for your comments.